glowing_waffles/coordinates.py

```diff
@@ -34,7 +34,7 @@ def convert_pixel_wcs(ccd_image, lon_or_ra, lat_or_dec, isPix=True):
     and the second lat/Dec.
 
     """
-    if isPix:
+    if is_pix:
         return ccd_image.wcs.all_pix2world(lon_or_ra, lat_or_dec, 0)
```

Comment:
Maybe you should add an argument to the function (with default value of 0) that can be passed in for the origin argument (which you now have set to 0).